pkg/asset/machines/openstack/machines.go

```diff
@@ -379,10 +373,8 @@ func failureDomainsFromSpec(mpool openstack.MachinePool) []machinev1.OpenStackFa
 	return failureDomains
 }
 
-func checkNetworkExtensionAvailability(cloud, alias string, opts *clientconfig.ClientOpts) (bool, error) {
-	if opts == nil {
-		opts = openstackdefaults.DefaultClientOpts(cloud)
-	}
+func CheckNetworkExtensionAvailability(cloud, alias string) (bool, error) {
```

Comment:
Linting is not happy about the lack of comment for a public function.

Reply:
yup, it's fixed on a force-push, thanks. PTAL